ts/features/itwallet/walletInstance/store/actions/index.ts

```diff
@@ -1,3 +1,4 @@
+import { WalletInstanceData } from "@pagopa/io-react-native-wallet/lib/typescript/client/generated/wallet-provider";
```

Comment:
Can we move this type in the itwTypesUtils.ts? We can extract it from the `getWalletInstanceStatus` method:

```typescript
export type WalletInstanceStatus = Awaited<
  ReturnType<typeof WalletInstance.getWalletInstanceStatus>
>;
```

Then we can use `WalletInstanceStatus` as a type for the revocation status

Reply:
Yes sure. Fixed in [19d9176](https://github.com/pagopa/io-app/pull/6547/commits/19d91762ee96d8ac5877ee1cf861126edbdb82d9)